distributed/worker_memory.py

```diff
@@ -102,6 +102,13 @@ def __init__(
         memory_pause_fraction: float | Literal[False] | None = None,
     ):
         self.logger = logging.getLogger("distributed.worker.memory")
+        for name, pattern in [
+            ("high-unmanaged", r"Unmanaged memory use is high"),
+            ("pausing", r".*Pausing worker"),
+            ("resuming", r".*Resuming worker"),
```

Comment:
Are pausing/resuming actually spamming?

Reply:
They have a potential to since there's no hysteresis

Reply:
Sure. I'm mostly wondering if this wouldn't be a case we actually want to see this oscillating. IMO this would not be very healthy and something is wrong. I also would hope that this oscillation would only last for a couple hundred ms.
Anyhow, no strong opinion just wondering